desktop-src/FileIO/file-attribute-constants.md

```diff
@@ -70,7 +70,7 @@ Example taken from a [Windows Classic Sample](https://github.com/microsoft/Windo
 | <span id="FILE_ATTRIBUTE_EA"></span><span id="file_attribute_ea"></span><dl> <dt>**FILE\_ATTRIBUTE\_EA**</dt> <dt>262144 (0x00040000)</dt> </dl> | A file or directory with extended attributes.<br/><br/>**IMPORTANT:** This constant is for internal use only. |
 | <span id="FILE_ATTRIBUTE_PINNED"></span><span id="file_attribute_pinned"></span><dl> <dt>**FILE\_ATTRIBUTE\_PINNED**</dt> <dt>524288 (0x00080000)</dt> </dl> | This attribute indicates user intent that the file or directory should be kept fully present locally even when not being actively accessed. This attribute is for use with hierarchical storage management software. |
 | <span id="FILE_ATTRIBUTE_UNPINNED"></span><span id="file_attribute_unpinned"></span><dl> <dt>**FILE\_ATTRIBUTE\_UNPINNED**</dt> <dt>1048576 (0x00100000)</dt> </dl> | This attribute indicates that the file or directory should not be kept fully present locally except when being actively accessed. This attribute is for use with hierarchical storage management software. |
-| <span id="FILE_ATTRIBUTE_RECALL_ON_OPEN"></span><span id="file_attribute_recall_on_open"></span><dl> <dt>**FILE\_ATTRIBUTE\_RECALL\_ON\_OPEN**</dt> <dt>262144 (0x00040000)</dt> </dl> | This attribute only appears in directory enumeration classes (FILE\_DIRECTORY\_INFORMATION, FILE\_BOTH\_DIR\_INFORMATION, etc.). When this attribute is set, it means that the file or directory has no physical representation on the local system; the item is virtual. Opening the item will be more expensive than normal, e.g. it will cause at least some of it to be fetched from a remote store. |
+| <span id="FILE_ATTRIBUTE_RECALL_ON_OPEN"></span><span id="file_attribute_recall_on_open"></span><dl> <dt>**FILE\_ATTRIBUTE\_RECALL\_ON\_OPEN**</dt> <dt>2097152 (0x00200000)</dt> </dl> | This attribute only appears in directory enumeration classes (FILE\_DIRECTORY\_INFORMATION, FILE\_BOTH\_DIR\_INFORMATION, etc.). When this attribute is set, it means that the file or directory has no physical representation on the local system; the item is virtual. Opening the item will be more expensive than normal, e.g. it will cause at least some of it to be fetched from a remote store. |
```

Comment:
I'm looking at the source for winnt.h in the latest Windows SDK, and the original value appears to be correct.
![image](https://github.com/user-attachments/assets/aeaad5d3-dd2d-4828-8607-e94492dac913)

Reply:
Thanks for checking out. Regarding the code snippet you posted, it looks like a bug to me.